.bash_profile

```diff
@@ -35,8 +35,8 @@ elif [ -f /etc/bash_completion ]; then
 fi;
 
 # Enable tab completion for `g` by marking it as an alias for `git`
-if type __git_wrap__git_main &> /dev/null; then
-    complete -o default -o nospace -F __git_wrap__git_main g
+if type __git_complete &> /dev/null; then
+    __git_complete g git
```

Comment:
I think the correct usage is `__git_complete g __git_main` since `git` is already an alias. Was running into problems with this with `__git_complete g git`

https://github.com/git/git/blob/master/contrib/completion/git-completion.bash#L3539